CMakeLists.txt

```diff
@@ -337,7 +350,8 @@ IF(NOT CYCLUS_DOC_ONLY)
         "${SQLite3_INCLUDE_DIR}"
         "${HDF5_INCLUDE_DIRS}"
         "${Boost_INCLUDE_DIR}"
-        "${COIN_INCLUDE_DIRS}")
+        "${COIN_INCLUDE_DIRS}"
+        "${OpenMP_CXX_INCLUDE_DIRS}")
```

Comment:
Is this defined at all if we don't set the parallel flag? Is that a problem?

Reply:
It won't be defined at all, but it shouldn't be a problem.  It will just append an empty string to the end of `inc_dirs`, which would happen when we build without COIN as well.  We could add the optional dependencies within respective `IF(<dep found>)...ENDIF()` blocks to handle this more gracefully

Reply:
It's fine this way, just wanted to check